app/javascript/chat/RequestManager/PersonalInvitationSection.jsx

```diff
@@ -19,6 +19,7 @@ export const PersonalInvitationSection = ({
         userInvitations.map((userInvitation) => {
           return (
             <PendingInvitationListItem
+              key={`${userInvitation.channel_name}-${userInvitation.membership_id}`}
```

Comment:
Refactor (blocking): `channel_name` is adequate for the key, as the context of this rendered component is for a specific user.